src/server/endpoints/covidcast.py

```diff
@@ -50,17 +88,15 @@ def handle():
     fields_int = ["time_value", "direction", "issue", "lag"]
     fields_float = ["value", "stderr", "sample_size"]
     q.set_fields(fields_string, fields_int, fields_float)
-    q.set_order('signal', 'time_value', 'geo_value', 'issue')
+    q.set_order("signal", "time_value", "geo_value", "issue")
```

Comment:
then we also want to return it as an additional field?

Reply:
for completeness sake, we could also add the `geo_type` and `time_type` columns.

Reply:
oh geez yes, otherwise there's no way to tell the counties from the metro areas. before we merge this we'll want to check that the covidcast library won't choke on the extra keys in the output.

Reply:
we can do the same trick as before: if the compatibility mode is enabled, we don't transfer it

Reply:
I changed the code such that unless in compatibility mode, it will transfer the new fields. However, one has always the option to use the `fields=...` param to filter them out again